database/methods.js

```diff
@@ -4,20 +4,38 @@ const driver = neo4j.driver("bolt://localhost:7687", neo4j.auth.basic("neo4j", "
 
 const getSong = (song_id, res) => {
   var session = driver.session();
-  const cypher = 'MATCH (song:Song {id: {id} }) RETURN song';
   const params = {id: song_id};
-  console.log(params);
-  return session
-    .run(cypher, params)
+  
+  return Promise.all([
+    session.run('MATCH (song:Song {id: {id} }) RETURN song.name', params),
+    session.run('MATCH (song:Song {id: {id} })-[:APPEARS_ON]->(album:Album) RETURN album', params)
+  ])
+  .then(result => {
+    res.send(result);
+    session.close();
+  })
+  .catch(error => {
+    session.close();
+    throw error;
```

Comment:
You might need to res.end() if there's an error (maybe also set error status)

Reply:
Thank you!